generators/client/templates/angular/webpack/webpack.common.js.ejs

```diff
@@ -77,22 +77,6 @@ module.exports = (options) => ({
                 SERVER_API_URL: `''`
             }
         }),
-        new webpack.optimize.CommonsChunkPlugin({
-            name: 'polyfills',
-            chunks: ['polyfills']
-        }),
-        new webpack.optimize.CommonsChunkPlugin({
```

Comment:
How are the chunks produced now using the default setting?

Reply:
autoMAGICALly

Reply:
hi ! 
about CommonsChunkPlugin i saw that on webpack4 was replaced by splitChunks  can you tell us why u dont use it please? 
and entry file not exist used even is it mandatory to start webpack4 ??